### PR TITLE
Resolve Sync progress going above 100%

### PR DESCRIPTION
**Pull request by DebbieArita**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes https://github.com/opensrp/fhircore/issues/3536

**Engineer Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [ ] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/SyncStatusView.kt [hunk 1]
```diff
@@ -170,8 +170,9 @@ fun SyncBottomBar(
         contentAlignment = Alignment.Center,
       ) {
         val context = LocalContext.current
-        when (currentSyncJobStatus) {
-          is CurrentSyncJobStatus.Running -> {
+        when {
+          currentSyncJobStatus is CurrentSyncJobStatus.Running &&
+            appDrawerUIState.percentageProgress!! <= 100 -> {
```
Comment (pld) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/SyncStatusView.kt:175]: if the percent is >100 I think we still want to run the update, we want to change the percent to from >100 to 100

**Review by pld — CHANGES_REQUESTED** [review]
